Lib/fontTools/ttLib/tables/O_S_2f_2.py

```diff
@@ -508,11 +513,17 @@ def intersectUnicodeRanges(unicodes, inverse=False):
 	True
 	"""
 	unicodes = set(unicodes)
-	uniranges = _getUnicodeRangeSets()
-	bits = set([
-		bit for bit, unirange in enumerate(uniranges)
-		if not unirange.isdisjoint(unicodes) ^ inverse])
-	return bits
+	unicodesteps, unicodevalues = _getUnicodeRanges()
+	bits = set(unicodevalues[bisect.bisect(unicodesteps, code)] for code in unicodes)
+	for code in unicodes:
+		# The spec says that bit 57 ("Non Plane 0") implies that there's
+		# at least one codepoint beyond the BMP; so I also include all
+		# the non-BMP codepoints here
+		if 0x10000 <= code <= 0x110000:
```

Comment:
This should be `0x10000 <= code < 0x110000`.

Reply:
:+1: